src/app/Library/Uploaders/Validation/ValidUploadMultiple.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
![happy-the-office-gif-by-cameo](https://user-images.githubusercontent.com/1032474/233600568-0840ed37-d901-4adc-9fb6-855735d9824b.gif)